src/app/api/services/dot-http-error-manager/dot-http-error-manager.service.ts

```diff
@@ -0,0 +1,105 @@
+import { DotRouterService } from '../dot-router/dot-router.service';
+import { DotMessageService } from '../dot-messages-service';
+import { Injectable } from '@angular/core';
+
+import { Observable } from 'rxjs/Observable';
+
+import { ResponseView, LoginService, HttpCode } from 'dotcms-js/dotcms-js';
+
+import { DotDialogService } from '../dot-dialog';
+
+/**
+ * Handle the UI for http errors messages
+ *
+ * @export
+ * @class DotHttpErrorManagerService
+ */
+@Injectable()
+export class DotHttpErrorManagerService {
+    constructor(
+        private dotDialogService: DotDialogService,
+        private dotMessageService: DotMessageService,
+        private loginService: LoginService,
+        private dotRouterService: DotRouterService
+    ) {
+        this.dotMessageService
+            .getMessages([
+                'dot.common.http.error.403.header',
+                'dot.common.http.error.403.message',
+                'dot.common.http.error.500.header',
+                'dot.common.http.error.500.message'
+            ])
+            .subscribe();
+    }
+
+    /**
+     * Handle the http error message and return a true if it did a redirect
+     *
+     * @param {ResponseView} err
+     * @returns {Observable<boolean>}
+     * @memberof DotHttpErrorManagerService
+     */
+    handle(err: ResponseView): Observable<any> {
```

Comment:
Not sure is this is the best approach to handle "forbidden" requests, but I don't have much coming from the server, this is still WIP.